recipes/dexplot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "dexplot" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0a9aee9685a2a69f4243684c75077a7cea012973c4cff4ce0a661a5c880973a2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - numpy >=1.15.0
+    - matplotlib-base >=3.1.0
+    - pandas >=0.24.0
+    - scipy >=1.0.0
+
+test:
+  imports:
```

Comment:
a thing we've been starting to put in recently: as those dependencies (and _their_ dependencies) are not particularly lightweight, adding this:

```yaml
test:
  imports:
    - dexplot
  requires:
    - pip
  commands:
    - python -m pip check
```

will ensure that _at least_ `pip` thinks your environment is self-consistent. yet another plug for `grayskull`: it adds this, or equivalent.